pages/index.js

```diff
@@ -1,21 +1,33 @@
 import Head from "next/head";
 import styled from "styled-components";
 import { Inter } from "next/font/google";
+import { useRouter } from "next/router";
+import useSWR from "swr";
 
 const inter = Inter({ subsets: ["latin"] });
 
-export default function Home() {
+export default function Home({ name, location, image, bewerten, preis }) {
+  const router = useRouter();
+
   return (
     <>
       <Head>
         <title>Capstone Project</title>
-        <meta name="description" content="Penguin Capstone Project" />
+        <meta name="description" content="Capstone Project: Melodic Fever" />
         <meta name="viewport" content="width=device-width, initial-scale=1" />
         <link rel="icon" href="/favicon.ico" />
       </Head>
       <main className={inter.className}>
-        <Heading>🐧Penguin Capstone Template🐧</Heading>
+        <Heading>Melodic Fever</Heading>
```

Comment:
why heading and not h1?
is Heading defined somewhere? :)

Reply:
☝🏻 good point!